src/test/java/com/spotify/folsom/MemcacheClientBuilderTest.java

```diff
@@ -47,36 +47,51 @@ public void tearDown() throws Exception {
   public void testValidLatin1() throws Exception {
     AsciiMemcacheClient<String> client = MemcacheClientBuilder.newStringClient()
             .withKeyCharset(Charsets.ISO_8859_1)
-            .withAddress(HostAndPort.fromParts("localhost", server.getPort()))
+            .withAddress(HostAndPort.fromParts("127.0.0.6", server.getPort()))
```

Comment:
Why are we using 127.0.0.6, as opposed to 127.0.0.1? For TCP tuple uniqueness? Shouldn't the port already solve that? Seems a bit messy to have to ensure unique IPs in tests.

Reply:
I wanted to simplify associated log lines with test code - ports are usually dynamic